apps/nextra/pages/en/build/sdks/ts-sdk/legacy-ts-sdk/typescript-sdk-overview.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: "TypeScript SDK Architecture"
+slug: "typescript-sdk-overview"
```

Comment:
drop the slug, since it's the same as the filename.  these end up causing some annoying troubles later